src/main/scala/strawman/collection/Iterable.scala

```diff
@@ -32,6 +32,8 @@ trait IterableOps[+A, +CC[X], +C] extends Any {
 
   protected[this] def coll: Iterable[A]
 
+  protected[this] def c: C
```

Comment:
I add this member to `IterableOps[A, CC[_], C]`. Why is it useful? In this PR it’s needed by `WithFilter`, which uses a `BuildFrom` instance to build the result of `map` and `flatMap`, and `BuildFrom` needs an initial collection `C` to create a builder.

Note, that this member was previously already existing in `SeqOps`. It was needed because `SeqOps` is extended by (at least) `StringOps` and `List`, and to implement methods of `SeqOps` we sometimes needed a `C` (ie. `String` or `List[A]`) value and sometimes an `Iterable[A]` (ie. `StringView` or `List[A]`) value. So, with just one member (`coll`) we couldn’t get both a `String` and an `Iterable[Char]`, that’s why we needed two members: `coll: Iterable[A]` and `c: C`.